src/components/notebook/DebugPanel.tsx

```diff
@@ -300,4 +301,16 @@ const DebugPanel: React.FC<DebugPanelProps> = ({
   );
 };
 
+function DebugPin() {
+  const firstItem = useQuery(queryDb(tables.debugPin.select()));
```

Comment:
This should be `tables.debug` now right? 

I think you'll want to change the `package.json` back to `"@runt/schema": "jsr:^0.6.3",` when you get a chance.